docs/_data/categories_thing.csv

```diff
@@ -11,20 +11,21 @@ HVAC,"Air condition devices, Fans",
 Inverter,"Power inverter, such as solar inverters etc.",
 LawnMower,"Lawn mowing robots, etc.",lawnmower.png
 Lightbulb,"Devices that illuminate something, such as bulbs, etc.",lightbulb.png
-Lock,"Devices whose primary pupose is locking something",lock.png
+Lock,"Devices whose primary purpose is locking something",lock.png
 MotionDetector,"Motion sensors/detectors",
 NetworkAppliance,"Bridges/Gateway need to access other devices like used by Philips Hue for example, Routers, Switches",
 PowerOutlet,"Small devices to be plugged into a power socket in a wall which stick there",poweroutlet.png
 Projector,"Devices that project a picture somewhere",projector.png
 RadiatorControl,"Controls on radiators used to heat up rooms",
-Receiver,"Audio/Video receivers, i.e. radio receivers, satelite or cable receivers, recorders, etc.",receiver.png
+Receiver,"Audio/Video receivers, i.e. radio receivers, satellite or cable receivers, recorders, etc.",receiver.png
 RemoteControl,"Any portable or hand-held device that controls the status of something, e.g. remote control, keyfob etc."
 Screen,"Devices that are able to show a picture",screen.png
 Sensor,"Device used to measure something",
 Siren,"Siren used by Alarm systems",siren.png
 SmokeDetector,"Smoke detectors",
 Speaker,"Devices that are able to play sounds",
-WallSwitch,"Any device attached to the wall that controls the status of something, for ex. a light switch, dimmer",wallswitch.png
+WallSwitch,"Any device attached to the wall that controls the status of something, e.g. a light switch, dimmer",wallswitch.png
+Water,"Devices used to control water. e.g. a flow stop valve.",water.png
```

Comment:
IIRC the thing categories are primarily meant to help UIs to generically render some nice little icons. I always think of then like "how things look like". And for that I reckon "water" would not be such a good candidate. 
Something like "Valve" however would be more easy to depict, I guess. Maybe it's too specific (@kaikreuzer ?) but I would rather fancy "Valve" instead of "Water".  What do you think?